packages/cli/src/lib/webpack/prerender.js

```diff
@@ -37,7 +37,7 @@ module.exports = function (env, params) {
 			throw err;
 		}
 
-		handlePrerenderError(err, env, stack, entry);
+		await handlePrerenderError(err, env, stack, entry);
```

Comment:
The end result of a prerender error is a good ol' `process.exit(1)`, so we want to block Webpack if we're at this point; there's no point in allowing it to continue whilst we track down the error position.